holoviews/util.py

```diff
@@ -69,15 +72,17 @@ def _dynamic_operation(self, map_obj):
             def dynamic_operation(*key, **kwargs):
                 self.p.kwargs.update(kwargs)
                 return self._process(map_obj[key], key)
-            return dynamic_operation
-
-        def dynamic_operation(*key, **kwargs):
-            key = key[0] if map_obj.mode == 'open' else key
-            self.p.kwargs.update(kwargs)
-            _, el = util.get_dynamic_item(map_obj, map_obj.kdims, key)
-            return self._process(el, key)
-
-        return dynamic_operation
+        else:
+            def dynamic_operation(*key, **kwargs):
+                key = key[0] if map_obj.mode == 'open' else key
+                self.p.kwargs.update(kwargs)
+                _, el = util.get_dynamic_item(map_obj, map_obj.kdims, key)
+                return self._process(el, key)
+        if isinstance(self.p.operation, ElementOperation):
```

Comment:
It is never used, but is useful information and will in future be used to look up the ``outputs``.